### PR TITLE
Cyphort: Community app release

### PR DESCRIPTION
**Pull request by sahils-crest**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [ ] Testing of all the changes has been performed (for bug fixes / features)
- [ ] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [ ] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [ ] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ ] Do NOT submit updates to dependencies unless it fixes an issue.
- [ ] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: cyphort.json [hunk 1]
```diff
@@ -0,0 +1,826 @@
...
+    "type": "sandbox",
+    "main_module": "cyphort_connector.py",
+    "app_version": "2.0.0",
+    "utctime_updated": "2018-12-07T01:05:47.000000Z",
+    "package_name": "phantom_cyphort",
+    "product_vendor": "Cyphort",
+    "product_name": "Cyphort",
+    "product_version_regex": ".*",
+    "min_phantom_version": "5.0.0",
+    "publisher": "Splunk",
```
Comment (dgopani-crest) [inline cyphort.json]: It should be Splunk Community

File: cyphort.json [hunk 2]
```diff
@@ -0,0 +1,826 @@
...
+                },
+                "file_name": {
+                    "description": "Filename to use",
+                    "verbose": "This value will be passed to the sandbox as the name of the file that is picked up up from the vault.",
+                    "data_type": "string",
+                    "order": 1,
+                    "contains": [
+                        "file name"
+                    ],
+                    "required": false
```
Comment (dgopani-crest) [inline cyphort.json]: We can remove this

File: release_notes/unreleased.md [hunk 3]
```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Release on splunk soar [PAPP-21321]
```
Comment (dgopani-crest) [inline release_notes/unreleased.md]: Update with correct sentence for example "Releasing as open-source Community app"

**Comment by dgopani-crest** [comment]
@sahils-crest, Add existing release notes from the GitLab folder and add version-wise MD files for all the released versions. For example https://github.com/splunk-soar-connectors/alertfind/pull/2/files

**Comment by dgopani-crest** [comment]
@sahils-crest Add "python_version": "2.7" in JSON file

File: release_notes/release_notes.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!-- File: release_notest.html
```
Comment (dgopani-crest) [inline release_notes/release_notes.html]: License not required in this file. please see https://github.com/splunk-soar-connectors/cyphort/pull/1#issuecomment-981576795

**Review by dgopani-crest — DISMISSED** [review]

**Review by brian-phantom — APPROVED** [review]

**Review by gary-phantom — APPROVED** [review]

**Review by pdesai-crest — APPROVED** [review]

**Pull request merged**